packages/manager/modules/sms/src/sms/components/sms-composition/sms-composition.controller.js

```diff
@@ -12,16 +13,20 @@ export default class SmsCompositionController {
   }
 
   getMessageDetails(noStopValue) {
-    const isShort =
-      this.model.sender &&
-      SmsCompositionController.isShortNumber(this.model.sender)
-        ? true
-        : this.model.noStopClause;
-    const suffix = noStopValue !== undefined ? noStopValue : isShort;
-    this.model.messageDetails = this.TucSmsMediator.getSmsInfoText(
-      this.model.message,
-      !suffix,
-    );
+    // timeout is here because the onChange event of the oui-textarea
+    // is triggered before the model is updated
```

Comment:
Isn't the new model the value passed as parameter ?

Reply:
yes you're right, the issue here is that this method is also used as a callback for the on-change method on some radio-buttons in the template, so changing the method signature (textarea value as parameter) will conflict with the behavior of the radio buttons that are passing their model value as parameter